extensions/sql-migration/src/wizard/accountsSelectionPage.ts

```diff
@@ -80,6 +97,50 @@ export class AccountsSelectionPage extends MigrationWizardPage {
 		};
 	}
 
+	private createAzureTenantContainer(view: azdata.ModelView): azdata.FormComponent {
+
+		const azureTenantDropdownLabel = view.modelBuilder.text().withProps({
+			value: constants.AZURE_TENANT,
+			CSSStyles: {
+				'margin': '0px'
+			}
+		}).component();
+
+		this._accountTenantDropdown = view.modelBuilder.dropDown().withProps({
+			width: WIZARD_INPUT_COMPONENT_WIDTH
+		}).component();
+
+		this._accountTenantDropdown.onValueChanged(value => {
+			/**
+			 * Replacing all the tenants in azure account with the tenant user has selected.
+			 * All azure requests will only run on this tenant from now on
+			 */
+			if (value.selected) {
+				this.migrationStateModel._azureAccount.properties.tenants = [this.migrationStateModel.getTenant(value.index)];
+			}
+		});
+
+		this._accountTenantFlexContainer = view.modelBuilder.flexContainer()
+			.withLayout({
+				flexFlow: 'column'
+			})
+			.withItems([
+				azureTenantDropdownLabel,
+				this._accountTenantDropdown
+			], { CSSStyles: { 'margin': '2px', } })
```

Comment:
seems a little painful to have to provide margin styles on dropdown boxes.  why aren't the default values good enough (i.e. should we change the default margins to look better or is there something special with the styles in this case).

Reply:
I will remove this. This does not impact the visuals a lot. 
I was trying something out with the margins and kept the same everywhere